electron/legendary_utils/user.ts

```diff
@@ -0,0 +1,33 @@
+import { exec } from 'child_process'
+import {
+  existsSync,
+  readFileSync
+} from 'graceful-fs'
+
+import { UserInfo } from '../types'
+import {
+  legendaryBin,
+  userInfo
+} from '../constants'
+
+export class User {
+  public static login(sid: string) {
+    exec(`${legendaryBin} auth --sid ${sid}`)
+  }
+
+  public static logout() {
+    exec(`${legendaryBin} auth --delete`)
```

Comment:
I think to avoid a racing condition here. The ideal would be to use `await execAsync` on the first one.

Reply:
Alright.